### PR TITLE
feat:added login page

### PR DESCRIPTION
**Pull request by GuptaEesh**
Created a login page for EComm:
- Remember me check box.
- Forgot password reset
- create account option.

ecomm6

### CODE REVIEW

File: login/login.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    <div class="login-dialogue log-in glass">
+
+        <label>
+            <h2>Email Address</h2>
+            <input type="text" class="input-text md" placeholder="E-mail Address...">
+        </label>
+        <label>
+            <h2>Password</h2>
+            <input class="input-text check md" type="password" pattern="(?=.*\d)(?=.*[a-z])(?=.*[A-Z]).{4,}"
+                maxlength="15" placeholder="Password[A a .*\ minlen-4 ]" />
```
Comment (lannisterrr) [inline login/login.html:54]: placeholder should not contain regex code, instead check for error using JS.

Comment (GuptaEesh) [reply login/login.html:54]: It's just for user reference, it doesn't do any work there.
- Regex works in pattern attr of input.

**Review by AnuragSahu11 — APPROVED** [review]
Code and looks good, you can merge , just a small suggestion if you will give small space between the input label , and actual input then it will look better

**Comment by GuptaEesh** [comment]
Revisited styling for the login page. Fixed some of it. Merging the branch now.

**Pull request merged**